pyproject.toml

```diff
@@ -34,7 +34,7 @@ dependencies = [
     "importlib-metadata; python_version < '3.8'",
     "Jinja2>=3.0.0",
     "msgpack",
-    "packaging",
+    "packaging>=22.0",
```

Comment:
@pankajkoti would it make sense for us to add this restriction to the tests only?
I'm wondering if there are any circumstances when we still want users to be able to use older versions of the `packaging` package.

Reply:
It’s generally safe to add a `>=` constraint on `packaging` since it has a strict backward compatibility policy. I would suggest just adding the version range globally.